org.qutebrowser.qutebrowser.yml

```diff
@@ -0,0 +1,40 @@
+---
+app-id: org.qutebrowser.qutebrowser
+runtime: org.kde.Platform
+runtime-version: '5.12'
+sdk: org.kde.Sdk
+command: qutebrowser
+rename-desktop-file: qutebrowser.desktop
+rename-icon: qutebrowser
+finish-args:
+- "--share=ipc"
```

Comment:
Check the [YAML style guide](https://github.com/flathub/flathub/wiki/YAML-Style-Guide); the indenting should also apply for child elements.